src/core/Tm.ml

```diff
@@ -786,9 +786,9 @@ and pp_cmd env fmt (hd, sp) =
     | Snoc (sp, f)->
       match f with
       | Car ->
-        Format.fprintf fmt "@[<hv1>(car@ %a)@]" (go `Car) sp
+        Format.fprintf fmt "@[<hv1>%a .fst@]" (go `Car) sp
```

Comment:
I think this should be `(fst %a)`, since it's the core language. (Or else, we should change how the core language is parsed, which is another option.)

Reply:
I thought about it, and realize we should probably do a more thorough change with s/Cdr/Fst/ or something... I can do the Lisp style for now.